### PR TITLE
CI on GKE: Switch to logrus

### PR DESCRIPTION
**Pull request by spxtr**
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/test-infra/515)

### CODE REVIEW

File: ciongke/cmd/hook/kubeagent.go [hunk 1]
```diff
@@ -64,22 +64,33 @@ type kubeClient interface {
 // Cut off test-pr jobs after 10 hours.
 const jobDeadline = 10 * time.Hour
 
+func fields(kr KubeRequest) logrus.Fields {
```
Comment (apelisse) [inline ciongke/cmd/hook/kubeagent.go]: Would it make sense to make the following instead?
```
func (kr KubeRequest) field() logrus.Fields {
```

Comment (spxtr) [inline ciongke/cmd/hook/kubeagent.go]: I don't think so. I'm not trying to change the behavior of `KubeRequest`, but instead writing a pure function that takes one and turns it into the other. Presumably I could have multiple such functions for different log levels or something, but I don't actually.

**Comment by spxtr** [comment]
@apelisse gave me the OK

**Comment by apelisse** [comment]
Yes, lgtm

**Pull request merged**